Cesium.Compiler/FileSystemIncludeContext.cs

```diff
@@ -6,28 +6,39 @@ public class FileSystemIncludeContext : IIncludeContext
 {
     private readonly string _stdLibDirectory;
     private readonly string _currentDirectory;
+    private readonly List<string> visitedFiles = new();
```

Comment:
_(Minor)_ I suggest a rename:
```suggestion
    private readonly List<string> _guardedIncludedFiles = new();
```